snapcraft/internal/repo/_deb.py

```diff
@@ -380,6 +396,12 @@ def _manifest_dep_names(self, apt_cache):
         return manifest_dep_names
 
 
+def _find_virtual_package_provider(apt_cache, package_name):
```

Comment:
whaaat? I searched the api for something like this, and clearly did a lousy job. Thanks for finding it!